target/linux/realtek/rtl838x/config-5.10

```diff
@@ -7,8 +7,8 @@ CONFIG_BLK_DEV_RAM=y
 CONFIG_BLK_DEV_RAM_COUNT=16
 CONFIG_BLK_DEV_RAM_SIZE=4096
 # CONFIG_BMIPS_CPUFREQ is not set
-CONFIG_CEVT_R4K=y
-CONFIG_CEVT_RTL9300=y
+# CONFIG_CEVT_R4K is not set
+# CONFIG_CEVT_RTL9300 is not set
```

Comment:
I think `make kernel_menuconfig` actually deletes these.

btw, as discussed in a different PR, I think it's fair to also delete `patches-5.10/309-cevt-rtl9300-support.patch` as part of this series.